instituto-brasileiro-de-informacao-em-ciencia-e-tecnologia-abnt.csl

```diff
@@ -495,6 +589,28 @@
             <text macro="access"/>
           </group>
         </else-if>
+        <else-if type="graphic">
+          <group suffix=".">
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text variable="medium" suffix=", "/>
+            <text variable="dimensions" suffix=". "/>
+            <text variable="archive"/>
+            <text macro="access" prefix=". "/>
+          </group>
+        </else-if>
+        <else-if type="interview">
+          <group suffix=".">
+            <text macro="author" suffix=". "/>
```

Comment:
Throughout the style, use delimiters on the group, not suffixes, for this sort of punctuation.